script.js

```diff
@@ -0,0 +1,114 @@
+const colors = [
+    "#0fbcf9",
+    "#00d8d6",
+    "#05c46b",
+    "#ffc048",
+    "#ffdd59"
+  
+    ];
+
+  
+    let firstColor = colors[Math.floor(Math.random() * colors.length)];
+    let secondColor = colors[Math.floor(Math.random() * colors.length)];
+    
+    document.body.style.background = `linear-gradient(${firstColor}, ${secondColor})`;
+   
+    const toDoButton = document.getElementById("toDoButton");
+    const toDo = document.getElementById("toDo");
+    const toDoList = document.getElementById("todo-list");
+    const toDoInfo = document.getElementById("toDoInfo");
+    const doneList = document.getElementById("done-list");
+    const doneInfo = document.getElementById("doneInfo");
+    const finishtoDo = document.getElementById("finishtoDo");
+    
+    let toDoNum = 0;
+    let doneNum = 0;
+
+    function finish(){
+        if(toDoNum === 0){
+          finishtoDo.classList.remove("hidden");
+    
+        }
+    
+        else{
+          finishtoDo.classList.add("hidden");
+        }
+    
+       
+      }
+
+    function deleteToDo(event) {
+        toDoNum = toDoNum =< 0 ? 0 : toDoNum--
+        toDoInfo.innerText =`(${toDoNum})`;
+        
+        const deletetoDoList = event.target.parentElement;
+        deletetoDoList.remove();
+      
+        finish();
+
+        }
+
+        function moveToDo(event) {
+            deleteToDo(event);
+            
+            const removeList = document.createElement("div");
+            const removeItem = event.target;
+            const deleteButton = document.createElement("span");
+            deleteButton.innerText = "❌";
+           
+            removeList.style.color ="grey";
+            
+            removeList.appendChild(removeItem); 
+            removeList.appendChild(deleteButton);
+            doneList.appendChild(removeList);
```

Comment:
지금 상황에선 채림 님 말씀처럼 `moveToDo` 함수를 아래와같이 작성하면 삭제 버튼을 또 만들 필요는 없어 보입니다

```suggestion
function moveToDo(event) {
  deleteToDo(event);

  const removeList = document.createElement('div');
  const removeItem = event.target.parentElement;

  removeList.style.color = 'grey';

  removeList.appendChild(removeItem);
  doneList.appendChild(removeList);

  doneNum++;
  doneInfo.innerText = `(${doneNum})`;
}
```